### PR TITLE
WIP-life-insurance-parity edits

### PR DESCRIPTION
**Pull request by peggygannon**
## Page to edit
- [ ] https://www.va.gov/life-insurance/options-eligibility/sgli/
- [ ] https://www.va.gov/life-insurance/manage-your-policy/
- [ ] https://www.va.gov/life-insurance/options-eligibility/
- [ ] https://www.va.gov/life-insurance/options-eligibility/vgli/
- [ ] https://www.va.gov/life-insurance/options-eligibility/fsgli/

## Origin of request (internal/stakeholder/user feedback)
Life insurance parity edits

## Description of what's needed (edits/link changes/additions)
Assess and implement changes to bring parity between new VA.gov pages and legacy VA.gov pages in the Life Insurance section.

### CODE REVIEW

**Comment by peggygannon** [comment]
@bethpotts Life Insurance parity PR is ready for your review.

File: pages/life-insurance/assess-life-insurance-needs.md [hunk 1]
```diff
@@ -1,7 +1,7 @@
 ---
 order: 1
-spoke: More resources
-href: https://www.benefits.va.gov/insurance/lifeins101.asp
-title: Assess Your Life Insurance Needs
+spoke: More Resources
```
Comment (ncksllvn) [inline pages/life-insurance/assess-life-insurance-needs.md]: Heads-up that I think that this needs to be sentence case 👀

Comment (peggygannon) [reply pages/life-insurance/assess-life-insurance-needs.md]: @ncksllvn Ok, thanks. this looks like a new page that was added to the PR. Whoever set it up probably copied a template from before we sentence-cased. Should I go in and fix this now?

**Comment by peggygannon** [comment]
@ncksllvn This PR is ready for your review and then merge. I tried to merge in the master branch, but when I clicked the button above it said "Update branch attempt failed"

**Pull request merged**